.gitignore

```diff
@@ -21,3 +21,6 @@ release/
 .cargo-ok
 Cargo.toml.bak
 expand.rs
+
+# Js
+**/node_modules/
```

Comment:
It could (not 100% sure) work without the leading `**/`

Reply:
Indeed, it works well without `**/`.